src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Msmq/MsmqCommon.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Text;
+using Datadog.Trace.ClrProfiler.Integrations;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Msmq
+{
+    internal class MsmqCommon
+    {
+        internal const string IntegrationName = nameof(IntegrationIds.Msmq);
+        internal static readonly IntegrationInfo IntegrationId = IntegrationRegistry.GetIntegrationInfo(IntegrationName);
+
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(MsmqCommon));
+
+        internal static Scope CreateScope(Tracer tracer, string command, string spanKind, string queueName, string formatName, string queueLabel, DateTime queueLastModifiedTime, bool withinTransaction, string transactionType, bool transactionalQueue, out MsmqTags tags)
```

Comment:
This is a matter of preference, but could we pass an `IMessageQueue` in here instead of the strings? Also I don't think the out parameter is used anywhere?